examples/prediction_upload/video_predictions.ipynb

```diff
@@ -64,18 +64,7 @@
         "!pip install -q 'labelbox[data]'"
```

Comment:
<p>Probably good to remove this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1006/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>